R/tate_text.R

```diff
@@ -0,0 +1,22 @@
+#' Tate Gallery modern artwork metadata
+#'
+#' Metadata such as artist, title, and year created for recent artworks owned
+#'  by the Tate Gallery. Only artworks created after 1980 are included, and the
+#'  metadata source was last updated in 2014. Tate does not endorse the
+#'  extracted data set here in this R package.
```

Comment:
I'm a little confused about this sentence:

> Tate does not endorse the extracted data set here in this R package.

Does it mean that Tate doesn't endorse the data being in this package, or that Tate doesn't endorse the way it was extracted?

Reply:
I was trying to be extra careful about aligning with this:
https://github.com/tategallery/collection#usage-guidelines-for-open-data
Especially this part:
https://github.com/tategallery/collection#be-responsible

Any thoughts on how to be clear with this, or whether it is necessary in this context?

Reply:
Maybe it is okay. All this legal stuff is hard to read anyways.

Maybe something along the lines of

> Tate doesn't endorse you or your use of the data set unless you have prior permission to do so.

Reply:
How about 

> The Tate Gallery provides these data but would like users to be respectful of their [guidelines for use](https://github.com/tategallery/collection#usage-guidelines-for-open-data).